content/en/developers/dogstatsd/high_throughput.md

```diff
@@ -218,6 +224,83 @@ Then set the Agent `dogstatsd_so_rcvbuf` configuration option to the same number
 dogstatsd_so_rcvbuf: 26214400
 ```
 
+### Over UDS (Unix Domain Socket)
+
+#### Linux
+
+For UDS sockets, Linux is internally buffering datagrams in a queue if the
+reader is slower than the writer. The size of this queue represents the maximum number
+of datagrams that Linux will buffer per socket. This value can be queried with the following command:
+
+```bash
+sysctl net.unix.max_dgram_qlen
+```
+
+If the value is < 512, you can increase it to 512 or more using this command:
+
+```bash
+sysctl -w net.unix.max_dgram_qlen=512
+```
+
+Add the following configuration to `/etc/sysctl.conf` to make this change permanent:
+
+```conf
+net.unix.max_dgram_qlen = 512
+```
+
+In the same manner, the `net.core.wmem_max` could be incremented to 4MiB to improve
+client writing performances:
+
+```conf
+net.core.wmem_max = 4194304
+```
+
+Then set the Agent `dogstatsd_so_rcvbuf` configuration option to the same number in `datadog.yaml`:
+
+```yaml
+dogstatsd_so_rcvbuf: 4194304
+```
+
+Please note that setting these values will need a reboot of the host.
+
+### Ensure proper packet sizes
+
+Sending packets with an adequate size to Dogstatsd will avoid extra work to the server.
+
+If the packets sent are too small, Dogstatsd will have to pack several together in order
+to process them in batches later in the pipeline. The official Dogstatsd clients are
+capable of grouping metrics to have the best ratio of the number of metrics per packet.
+
+If the clients properly send packets having a size close to `dogstatsd_buffer_size`, Dogstatsd
+will perform better. Be careful: the packets must not be larger than this size, otherwise, the
+Dogstatsd server won't be able to load them completely in the buffer and some of metrics will
+be malformed. Please use the corresponding configuration field in your Dogstatsd clients.
+
+Note for UDP: as the UDP packets are most of the time going through the Ethernet and IP layer,
+they inherently have a size limited to 1472.
```

Comment:
1472 is the "usual limit" (IPv4 header with no option/1500 bytes Ethernet MTU) and it may be larger (bigger MTU) or smaller (due to IPv6 larger header).
```suggestion
Note for UDP: as the UDP packets are most of the time going through the Ethernet and IP layer, fragmentation should be avoided even if supported. Packet size should be limited to fit at most a single Ethernet frame and prevent IP fragmentation.
```
I'm not sure the suggested wording is understandable, but I think we should definitely mentioned that IP fragmentation should be avoided.

Reply:
You're right I'll add more precision here

Reply:
I've addressed your feedback, please re-open if you think that's not clear enough!